app/src/main/java/de/tum/in/tumcampusapp/component/ui/overview/MainFragment.kt

```diff
@@ -160,6 +160,11 @@ class MainFragment : BaseFragment<Unit>(
         super.onDestroy()
     }
 
+    override fun onDestroyView() {
+        cardsRecyclerView.adapter = null
```

Comment:
With the latest changes from #1417 this only compiles this way:

```suggestion
        binding.cardsRecyclerView.adapter = null
```
Which leads to this error though
```
java.lang.IllegalStateException: Should not attempt to get bindings when Fragment views are destroyed.
```

Can you check out how to handle this with viewBindings?